src/server/controllers/reaction.controller.ts

```diff
@@ -159,6 +161,27 @@ export const toggleReactionHandler = async ({
   input: ToggleReactionInput;
 }) => {
   try {
+    if (input.entityType === 'image') {
+      // We are only doing this for image contests for now. Because the user
+      // gets an instant feedback, this shouldn't have too bad an impact if any at all.
+      const contests = await getContestsFromEntity({
+        entityType: input.entityType,
+        entityId: input.entityId,
+      });
+
+      if (
+        contests.find(
+          (contest) =>
+            !!contest.metadata?.votingPeriodStart &&
+            dayjs(contest.metadata?.votingPeriodStart).isAfter(dayjs())
```

Comment:
There's a helper method for this already
```suggestion
            isFutureDate(contest.metadata.votingPeriodStart)
```